.github/pr_processor.py

```diff
@@ -79,7 +88,7 @@ def user_comment(issue, user=GITHUB_USER):
             for issue_num in r_issues:
                 r_issue = redmine.issue.get(issue_num)
 
-                if r_issue.tracker.name == "Issue":
+                if r_issue.tracker.name == "Issue" and repo not in TOOLS:
```

Comment:
If these repos don't have cherry pick tags, then it should just ignore them.

Reply:
See https://github.com/pulp/pulp-ci/blob/master/.github/pr_processor.py#L98-L102

Reply:
cool, I didn't notice it before, thanks!